CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## v0.7.0-beta (04/09/2019)
+
+**Common**
+
+* New abstract TaskScheduler introduced to be used for async context
+  (default thread pool implementation provided).
+* Added UserAgent to mark all triggered requests.
+* China Lookup API URL added.
+* New HTTP abstraction layer introduced without singleton usage.
+
+**olp-edge-sdk-cpp-authentication**
+
+* Use new HTTP layer as Settings input parameter.
+* TaskScheduler now as Settings input parameter.
+* Removed raw std::thread() usage and switched to TaskScheduler for callbacks.
+
+**olp-edge-sdk-cpp-dataservice-read**
+
+* Use new HTTP layer as OlpClientSettings input parameter.
+* TaskScheduler now as OlpClientSettings input parameter.
+* Removed raw std::thread() usage and switched to TaskScheduler for callbacks.
+* Improved logging.
```

Comment:
* Use TaskScheduler OlpClientSettings as an input parameter.
* Removed usage of raw `std::thread()` and switched to TaskScheduler for callbacks.
* Improved logging.